src/atomate2/aims/sets/base.py

```diff
@@ -0,0 +1,571 @@
+"""Module defining base FHI-aims input set and generator."""
+from __future__ import annotations
```

Comment:
Okay I can do this, I did not because the other codes also had sets defined here, and also in pymatgen so I was not sure where they belonged. I can fix this early next week when I have access to a properly setup computer again

Reply:
Yes, this is my fault. We are in the process of moving the VASP input sets to pymatgen and will work on moving the other codes afterwards.

Reply:
Okay then I will also make the move